db/1-schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS ACCOUNTS (
```

Comment:
We need to chat to discuss how to run "migrations" ... Ideally a schema change should be applied automatically with each new deploy. Maybe we're already set up for that?